docs/devel/integration.rst

```diff
@@ -55,17 +58,22 @@ translations to match your code base.
 Adding new strings
 ++++++++++++++++++
 
-In case your translation files are stored in VCS together with the code, you most
-likely have existing workflow for developers to introduce new strings. You
-might extend it by using :ref:`source-quality-gateway`.
+In case your translation files are stored in VCS together with the code,
+you most likely have existing workflow for developers to introduce new strings.
+You might extend it by using 
+
+Any way of adding strings will be picked up, but consider using
+:ref:`source-quality-gateway` to avoid also introducing errors.
 
 When the translation files are separate, there needs to be a way to introduce
```

Comment:
separate from?

Reply:
Not sure ... it probably should have been standalone. This paragraph talks about having translation files not in the VCS repository with the code and managing it externally.

Reply:
```suggestion
When the translation files are not in a upstream repository, there needs to be a way to introduce
```